benches/nexmark/main.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 the DBSP contributors.
+// SPDX-License-Identifier: MIT
+
+//! Nexmark provides a configurable Nexmark data source.
+//!
+//! Based on the API defined by the [Java Flink NEXMmark implementation](https://github.com/nexmark/nexmark)
+//! with some inspiration from the [Megaphone Nexmark tests](https://github.com/strymon-system/megaphone/tree/master/nexmark)
+//! which are themselves based on the [Timely Nexmark benchmark implementation](https://github.com/Shinmera/bsc-thesis/tree/master/benchmarks).
+//!
+//! I'm writing this Nexmark data source generator with the intention of
+//! moving it to its own repo and publishing as a re-usable crate, not only
+//! so we can work with the wider community to improve it, but also because
+//! it will allow us later to extend it to support multiple data sources in
+//! parallel when DBSP can be scaled.
+#![feature(test)]
+
+extern crate test;
```

Comment:
We shouldn't need the test feature, [`criterion`](https://github.com/bheisler/criterion.rs) is much better

Reply:
woops, I think I had that originally here when I had some example tests when I began. Removed now.